lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -26,9 +27,23 @@ function M.make_attach(normal_mode_functions, submodule, keymap_modes)
           mapping_description = function_description .. " " .. query_metadata
         end
 
-        vim.keymap.set(keymap_modes, mapping, function()
-          require("nvim-treesitter.textobjects." .. submodule)[function_call](query)
-        end, { buffer = bufnr, silent = true, remap = false, desc = mapping_description })
+        local function fn()
+          if opts.repeatable then
+            _G._nvim_treesitter_textobject_last_function = function()
```

Comment:
not nice :cry:

Reply:
but https://github.com/lewis6991/gitsigns.nvim/blob/main/lua/gitsigns/repeat.lua does not work due to `-` in the name.